client/src/components/App/App.jsx

```diff
@@ -1,41 +1,99 @@
 import ReactDOM from 'react-dom';
 import PropTypes from 'prop-types';
-import { Box, Link } from '@material-ui/core';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import { Box, Button, Link, Typography } from '@material-ui/core';
 import CssBaseline from '@material-ui/core/CssBaseline';
 import Header from '../Header/Header';
 import HeroBar from '../HeroBar/HeroBar';
 import ThemeProvider from '../ThemeProvider';
 import Form from '../Form/Form';
 import illustration from '../../../images/vaccine-illustration.jpg';
+import BackButton from '../BackButton/BackButton';
+import DownloadButton from '../DownloadButton/DownloadButton';
+import FactList from '../FactList/FactList';
 
 const App = props => (
   <ThemeProvider>
     <CssBaseline />
     <Header />
-    <HeroBar />
-    <Box
-      display="flex"
-      flexDirection="row"
-      justifyContent="center"
-    >
-      <Box mt={11} mr={4} maxWidth="530px">
-        <Form />
-      </Box>
-      <Box mt={2} ml={6} display="flex" flexDirection="column" alignItems="center">
-        <img src={illustration} alt="Woman receiving a vaccine" style={{maxWidth: '750px'}}/>
-        <Link
-          href="https://www.vecteezy.com/free-vector/epidemic-prevention"
-          color="secondary"
-          target="_blank"
-          rel="noopener"
-        >
-          Epidemic Prevention Vectors by Vecteezy
-        </Link>
-      </Box>
-    </Box>
 
+    <Router>    
+      <HeroBar />
+
+      <Switch>
+        <Route exact path="/">
+          <Box
+            display="flex"
+            flexDirection="row"
+            justifyContent="center"
+          >
+            <Box mt={11} mr={4} maxWidth="530px">
+              <Form />
+            </Box>
+            <Box mt={2} ml={6} display="flex" flexDirection="column" alignItems="center">
+              <img src={illustration} alt="Woman receiving a vaccine" style={{maxWidth: '750px'}}/>
+              <Link
+                href="https://www.vecteezy.com/free-vector/epidemic-prevention"
+                color="secondary"
+                target="_blank"
+                rel="noopener"
+              >
+                Epidemic Prevention Vectors by Vecteezy
+              </Link>
+            </Box>
+          </Box>
+        </Route>
+
+        <Route exact path="/data-found">
+          <BackButton />
+          <Box
+            display="flex"
+            flexDirection="row"
+            justifyContent="center"
+            mt={20}
+            mb={20}
+          >
+            SMART HEALTH CARD WILL GO HERE
+          </Box>
+          <Box display="flex" flexDirection="row" justifyContent="center">
+            <DownloadButton>
+              Download SMART&reg; Health Card
+            </DownloadButton>
+            <DownloadButton>
+              Download Printable PDF
+            </DownloadButton>
+          </Box>
+          <Box display="flex" flexDirection="row" justifyContent="center" mt={2} mx={5}>
```

Comment:
Could we adjust maxwidth of the FactList and the Typography at the bottom so that the words are contained under the button and SMART Health Card, as it is in the mockup? This is not a huge deal for now, but I think it will matter more when we go to make the PDF downloadable.